app/models/katello/host_queue_element.rb

```diff
@@ -0,0 +1,5 @@
+module Katello
+  class HostQueueElement < Katello::Model
+    serialize :metadata, Hash
```

Comment:
what is this used for?

Reply:
specifically the ` serialize :metadata, Hash`

Reply:
That was something I thought I needed but do not. Gone now!